cluster/master/api_backend.go

```diff
@@ -390,10 +390,6 @@ func (s *QKCMasterBackend) GetTip() uint64 {
 	return s.rootBlockChain.CurrentBlock().NumberU64()
 }
 
-func (s *QKCMasterBackend) IsSyncIng() bool {
```

Comment:
move IsSyncing(), IsMining(), CurrentBlock(), GetSlavePoolLen() funcs to api_backend.go in master forder

Reply:
done